Classes/Event/EventMetadata.php

```diff
@@ -1,112 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-/**
- * The EventMetadata is a container for arbitrary metadata for Events.
- * This class is immutable.
- *
- * @api
- */
-class EventMetadata
```

Comment:
This class was cumbersome to work with..
Now we only have one place where metadata can be added (via the signal) so IMO we don't need the whole immutable thingy here.
We could still communicate the best practice of namespacing any metadata that is added